solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,190 @@
+"""Simulating an ATM. """
+
+import os
+import sys
+import json
+
+ATM_FILE_PLACE = 1
+COSTUMER_CHOICE_OPTIONS = "For checking balance press  1 \nFor cash " \
+                          "withdrawal press 2 \nFor cash deposit press 3 " \
+                          "\nFor change password " \
+                          "press 4 \nTo finis actions press 5 \n"
+
+
+def cash_withdrawal(costumer_id, costumers_data):
+    """Withdrawal from the ATM amount of money.
```

Comment:
first word should be command

Reply:
To clarify, he means that the first word of any function doc should be "present progressive." (this generally means a verb ending in an 's')

For some made up examples:
"""Checks that two numbers are equal"""
"""Sends a message to the server"""
"""Reformats the list to the standard format"""

Reply:
its correct except for the end with 's'
in the examples, it should be check, send or reformat